django/core/management/commands/sendtestemail.py

```diff
@@ -1,21 +1,35 @@
 import datetime
 import socket
 
+from django.conf import settings
 from django.core.mail import send_mail
 from django.core.management.base import BaseCommand
 
 
 class Command(BaseCommand):
     help = "Sends a test email to the email addresses specified as arguments."
+    missing_args_message = "You must specify some email recipients, or pass the --managers or --admin arguments."
 
     def add_arguments(self, parser):
-        parser.add_argument('email', nargs='+',
+        parser.add_argument('email', nargs='*',
             help='One or more email addresses to send the test mail to.')
+        parser.add_argument('--managers', action='store_true', dest='managers', default=False,
+            help='Send the test email to all addresses specified in settings.MANAGERS.')
+        parser.add_argument('--admins', action='store_true', dest='admins', default=False,
+            help='Send the test email to all addresses specified in settings.ADMINS.')
 
     def handle(self, *args, **kwargs):
+        recipients = set(kwargs['email'])
```

Comment:
I've now made it create a set at time of passing the argument to send_mail. How does that look?